content/docs/communities/administration/_index.md

```diff
@@ -0,0 +1,166 @@
+
+**Upgrading:**  
+Clients that [support upgrading rooms](#sending-room-upgrades) usually offer a button or similar in their UI when an upgrade is available.
+Sometimes the client UI might allow you to choose the target room version, but usually the default room version defined by the [homeserver](@/docs/matrix-concepts/elements-of-matrix/_index.md#homeserver) is used.
+
+If the option is missing unexpectedly, it is possible that the combination of your client's UI and your homeserver's configuration does not recommend an upgrade at this time.
+Read on to learn how to upgrade anyway.
+
+**Advanced upgrading:**  
+There are some reasons why your client might not be showing you the upgrading UI despite an upgrade being available, including:
+- Your homeserver implementation or admin doesn't recommend it, via the [homeserver's configuration](https://spec.matrix.org/latest/client-server-api/#mroom_versions-capability).
+- Your client only recommends upgrading from or between certain versions.
+
+In these cases you can use e.g. Element Web's `/upgraderoom <version>` chat command after [enabling the developer tools](https://docs.element.io/latest/element-support/frequently-asked-questions/?h=devtools#matrix-general).
+You can also downgrade and sidegrade in this way.
+
+**Limitations:**  
+While the [Matrix specification](https://spec.matrix.org/v1.15/client-server-api/#server-behaviour-19) is not overly strict about it, your homeserver will execute the room upgrade on a best effort basis, trying to create the new room with settings matching the old room.
+The decentralised nature of Matrix can lead to circumstances that cannot be covered automatically by your homeserver.
+We suggest considering the following recommendations to mitigate this.
+
+#### Recommendations
+
+This is a non-exhaustive list of recommendations to consider when planning a room upgrade.
+Which of these cases apply to your room depends on your specific setup, for example size of your community.
+
+**Choose the "right" account on the "right" homeserver** to execute your upgrade.  
+The choice of the account starting the upgrade implies the homeserver that will perform the upgrade.
+Here are some important factors to guide your choice:
+
+1. Upgrading a room means that the room members need to receive the info about the upgrade and then join the new room.
+   The homeserver performing the upgrade by necessity becomes the center of attention here for a while, acting as a gateway to the new room in a sense.
+   It thus makes sense to choose a homeserver that is fast and well connected so it can handle all the joins following the room upgrade itself well.
+   If your community is centered around a certain homeserver, it probably makes sense use that one.
```

Comment:
```suggestion
   The homeserver performing the upgrade acts as a gateway to the new room.
   You want to choose a homeserver that is fast and well connected so it can handle all the joins following the room upgrade.
   If your community is centers around a certain homeserver, it makes sense use that one.
```